### PR TITLE
[MEME PR for testmerging]Removes restrictions on macro and microbombs.

### PR DESCRIPTION
**Pull request by Shadowlight213**
Relives the glory(horror) of the weekend where double agents had macrobombs.

### CODE REVIEW

**Comment by Bawhoppen** [comment]
inb4 more misused meme PR tag

**Comment by GunHog** [comment]
>inb4 more misused meme PR tag

What is the proper use?

**Comment by Bawhoppen** [comment]
its for prs that literally have the word meme in them

**Comment by PKPenguin321** [comment]
no it's for joke PRs like "removes mekhi" and shit like that
also this PR is a bad

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -975,16 +975,16 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
...
+
 
 /datum/uplink_item/implants/macrobomb
 	name = "Macrobomb Implant"
 	desc = "An implant injected into the body, and later activated either manually or automatically upon death. \
 			Upon death, releases a massive explosion that will wipe out everything nearby."
 	item = /obj/item/weapon/storage/box/syndie_kit/imp_macrobomb
-	cost = 20
-	include_modes = list(/datum/game_mode/nuclear)
+	cost = 1
```
Comment (Iamgoofball) [inline code/modules/uplink/uplink_item.dm]: uh

Comment (PKPenguin321) [inline code/modules/uplink/uplink_item.dm]: meme pr confirmed

**Comment by Bawhoppen** [comment]
@PKPenguin321 thats what they want you to think PKP you've fallen into their jew hands

**Comment by Shadowlight213** [comment]
Guys. This is literally just for a !!FUN!! testmerge going on in sybil currently. Not for actual merging.

File: code/modules/uplink/uplink_item.dm (hunk 1 above)
Comment (Shadowlight213) [inline code/modules/uplink/uplink_item.dm]: Joke off of when goof has microbomb uplink item spawn macrobombs

**Comment by Shadowlight213** [comment]
Just so people don't try and merge this.

**Pull request closed without merging**